### PR TITLE
Fix check user order

### PR DESCRIPTION
**Pull request by ntquy**

### CODE REVIEW

**Comment by ntquy** [comment]
ready

File: resources/views/layout/cart.blade.php [hunk 1]
```diff
@@ -38,7 +38,9 @@
 			</table>
 
 		</div>
+		@if( Auth::check() )
```
Comment (BunnyPi04) [inline resources/views/layout/cart.blade.php]: convention
@if (Auth::check())

**Comment by BunnyPi04** [comment]
commented

**Comment by ntquy** [comment]
ready

**Pull request merged**